config.json

```diff
@@ -37,14 +37,17 @@
         {
             "type": "file",
             "source": "./python_requirements.txt",
-            "destination": "/home/ec2-user"
+            "destination": "/home/ec2-user/"
```

Comment:
Just a note these are actually broken now with CfnCluster release 1.0.0 (default user to login is now `centos`)